draft-ietf-cbor-packed.md

```diff
@@ -655,6 +655,52 @@ from the same place could be:
 ])
 ~~~
 
+## Record Function Tag {#record}
+
+Tag 114 ('r') defines the "record" function, which combines
+an array of keys with an array of values into a map.
+
+The record function expects an array as its left-hand side,
+whose items are treated as key items for the resulting map,
+and an array of equal or shorter length as its right-hand side,
+whose items are treated as value items for the resulting map.
+
+The map is constructed by grouping key and value items
+with equal position in the provided arrays into pairs that constitute the resulting map.
+
+If the matching value item for a given key item is either >undefined< (simple value 23)
```

Comment:
If we adopt these semantics for >undefined< here, we could also allow deleting keys from default maps when combining two maps via concatenation: Having >undefined< in a value overriding a default could simply remove the corresponding pair.

Reply:
I wrote this up in https://github.com/cbor-wg/cbor-packed/pull/13 now.